internal/services/datafactory/data_factory_trigger_schedule_resource.go

```diff
@@ -430,13 +464,17 @@ func resourceDataFactoryTriggerScheduleRead(d *pluginsdk.ResourceData, meta inte
 		}
 
 		if pipelines := scheduleTriggerProps.Pipelines; pipelines != nil {
-			if len(*pipelines) > 0 {
+			if len(*pipelines) == 1 {
```

Comment:
No need to add this, when Len(pipeline) > 1, set the first pipeline to `pipeline_name` and `pipeline_parameters`, and always set them to `pipeline` blocks.